includes/JWTHandler.php

```diff
@@ -27,17 +27,20 @@ public function __construct(
     public function preprocessRawJWTData(
         string $rawJWTData
     ): string {
-        if (!str_starts_with($rawJWTData, 'Bearer:')) {
+        // Checks if the $rawJWTData string starts with the literal strings "Bearer:" or "Bearer ".
+        // If it doesn't start with either, it logs an error message and returns an empty string.
+        if (strpos($rawJWTData, 'Bearer:') !== 0 && strpos($rawJWTData, 'Bearer ') !== 0) {
```

Comment:
Any reason to change from `str_starts_with` to `strpos`? In my opinion, the former is more clear. Plus, there should not be any whitespace before `Bearer`.

Reply:
I agree it is more clear, but AFAICT `str_starts_with` was added in PHP 8.0. I haven't tested it on our PHP `7.4.33` though I think it would fail. What version are you using?

https://php.watch/versions/8.0/str_starts_with-str_ends_with